cg/cli/workflow/balsamic/base.py

```diff
@@ -197,28 +197,26 @@ def start(
     dry_run: bool,
 ):
     """Start full workflow for case ID."""
+    analysis_api: BalsamicAnalysisAPI = context.obj.meta_apis["analysis_api"]
+    analysis_api.prepare_fastq_files(case_id=case_id, dry_run=dry_run)
     LOG.info(f"Starting analysis for {case_id}")
-    try:
```

Comment:
So the try is no longer needed due to the new pre checks? Or do we want to catch `AnalysisNotReady`?